troubadix/standalone_plugins/no_solution.py

```diff
@@ -275,7 +275,11 @@ def main():
 
     summary = check_no_solutions(files, milestones, arguments.snooze)
 
-    print_report(term, summary, arguments.threshold, root)
+    found_vts = sum(len(entries) for _, entries in summary)
+
+    print_report(term, summary, arguments.threshold, root, found_vts)
+
+    return 1 if len(found_vts) > 0 else 0
 
 
 if __name__ == "__main__":
```

Comment:
Maybe remove this one then?

Reply:
This breaks the VS code debugger on my side

Reply:
Ah, let's keep it then